spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do 
+  before(:each) do
+    @factory = FacilityFactory.new
+    @service = DmvDataService.new
+
+    @co_dmv_data = @service.co_dmv_office_locations
+    @ny_dmv_data = @service.ny_dmv_office_locations
+  end
+
```

Comment:
Make sure that you test all methods - looks like we are missing one for `format_address` and `format_hours`